pybedlite/bed_record.py

```diff
@@ -183,6 +182,26 @@ def bed_fields(self) -> List[str]:
             ),
         ]
 
+    @property
+    def reference_name(self) -> str:
```

Comment:
I could be convinced this should just be called `refname` to match `Interval`

Reply:
Yes!